text-ui-test/input.txt

```diff
@@ -0,0 +1 @@
+todo borrow book
```

Comment:
You can also consider testing with corner cases, such as _null_ or try inputs that don't fit your command format